model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/NamedTests.xtend

```diff
@@ -8,25 +8,387 @@ import org.eclipse.xtext.testing.validation.ValidationTestHelper
+         }
+        '''.parse => [
+            m | m.assertError(JsldslPackage::eINSTANCE.named, JslDslValidator.JAVA_BEAN_NAMING_ISSUE)
+        ]
+    }
+
+    @Test
+    @TestCase("TC021")
+    @Requirement(reqs = #[
+            "REQ-SYNT-001",
+            "REQ-SYNT-002",
+            "REQ-SYNT-003",
+            "REQ-SYNT-004",
+            "REQ-MDL-001",
+            "REQ-TYPE-002"
+    ])
+    def void testNamingOfEnumFiel() {
+        '''
+         model modelTC021;
+
+         enum Enum {
+             xA0 = 0;
+             xB1 = 1;
+         }
+        '''.parse => [
+            m | m.assertError(JsldslPackage::eINSTANCE.named, JslDslValidator.JAVA_BEAN_NAMING_ISSUE)
+        ]
+    }
+
+    @Test
+    @TestCase("TC021")
+    @Requirement(reqs = #[
+            "REQ-SYNT-001",
+            "REQ-SYNT-002",
+            "REQ-SYNT-003",
+            "REQ-SYNT-004",
+            "REQ-MDL-001",
+            "REQ-ENT-001"
+    ])
+    def void testNamingOfEntityType() {
+        '''
+         model modelTC021;
+
+         entity xEnt1 {
+         }
+        '''.parse => [
+            m | m.assertError(JsldslPackage::eINSTANCE.named, JslDslValidator.JAVA_BEAN_NAMING_ISSUE)
+        ]
+    }
+
+    @Test
+    @TestCase("TC021")
+    @Requirement(reqs = #[
+            "REQ-SYNT-001",
+            "REQ-SYNT-002",
+            "REQ-SYNT-003",
+            "REQ-SYNT-004",
+            "REQ-MDL-001",
+            "REQ-ENT-001",
+            "REQ-ENT-012"
+    ])
+    def void testNamingOfEntityTypeWithExtendAndAbstract() {
+        '''
+         model modelTC021;
+
+         entity Ent {
+         }
+
+         entity abstract xEnt1 extends Ent {
+         }
+        '''.parse => [
+            m | m.assertError(JsldslPackage::eINSTANCE.named, JslDslValidator.JAVA_BEAN_NAMING_ISSUE)
+        ]
+    }
+
+    @Test
+    @TestCase("TC021")
+    @Requirement(reqs = #[
+            "REQ-SYNT-001",
+            "REQ-SYNT-002",
+            "REQ-SYNT-003",
+            "REQ-SYNT-004",
+            "REQ-MDL-001",
+            "REQ-MDL-003",
+            "REQ-TYPE-001",
+            "REQ-TYPE-002",
+            "REQ-TYPE-003",
+            "REQ-TYPE-004",
+            "REQ-TYPE-005",
+            "REQ-TYPE-006",
+            "REQ-TYPE-007",
+            "REQ-TYPE-008",
+            "REQ-TYPE-009",
+            "REQ-ENT-001",
+            "REQ-ENT-002"
+    ])
+    def void testNamingOfEntityField() {
+        '''
+         model modelTC021;
+
+         import judo::types;
+
+         type string SType(min-size = 0, max-size = 10);
+         type numeric Integer(precision = 9, scale = 0);
+
+         enum Enum {
+             a0 = 0;
+             b1 = 1;
+         }
+
+         entity Ent1 {
+             field Boolean xBool;
+             field SType xString;
+             field Integer xNumeric;
+             field Enum xEnum;
+             field Date xDate;
+             field Timestamp xTimestamp;
+             field Time xTime;
```

Comment:
It should be checked in JSL syntax, but I don't think the field type makes any difference during validation. If so the test can be simplified.